### ISSUE TITLE
Hydration warning on `BFormGroup`'s `label`

### ISSUE THREAD

**Issue by jonkri** [bootstrap-vue-next/bootstrap-vue-next#2103]
### Describe the bug

I'm getting a hydration error when using `BFormGroup`'s `label` prop.

```
[Vue warn]: Hydration attribute mismatch on 
<legend id="__BVID__nVl6Nsv1N9i-0___BV__BV_label___" class="form-label bv-no-focus-ring col-form-label pt-0" tabindex="-1"> 
  - rendered on server: id="__BVID__nVl6Nsv1N9i-0___BV__BV_label___"
  - expected on client: id="__BVID__nceJz32Yti8_0___BV__BV_label___"
  Note: this mismatch is check-only. The DOM will not be rectified in production due to performance overhead.
  You should fix the source of the mismatch. 
  at <Anonymous id="b-form-group" label="Label" > 
  at <App key=4 > 
  at <NuxtRoot>
```

I created a new Nuxt 3 project and picked the default options.

```
$ npx nuxi@latest init example
$ cd example
```

I installed `bootstrap`, `bootstrap-vue-next` and `@bootstrap-vue-next/nuxt`:

```
$ npm install --save-dev bootstrap bootstrap-vue-next @bootstrap-vue-next/nuxt
```

I added `modules: ['@bootstrap-vue-next/nuxt'],` and `css: ['bootstrap/dist/css/bootstrap.min.css'],` to `nuxt.config.ts`.

Finally, I replaced the code in `app.vue` with:

```html
<template>
  <BFormGroup id="b-form-group" label="Label" />
</template>
```

### Reproduction

[bootstrap-vue-next-issue-2103.zip](https://github.com/user-attachments/files/16569264/bootstrap-vue-next-issue-2103.zip)

### Used Package Manager

npm

### Validations

- [X] Have tested with the latest version. This is still alpha version and sometime things change rapidly.
- [X] Follow our [Code of Conduct](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by jonkri** [comment]
Still getting hydration warnings.

**Comment by codetheorist** [comment]
I'm getting these too, caused by differing IDs in the form group label.

**Comment by VividLemon** [comment]
Dont pass zip files as reproductions please

**Comment by VividLemon** [comment]
Should be fixed when using Vue 3.5 `useId`
